clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerServiceDelegator.java

```diff
@@ -319,19 +324,23 @@ public class CurrentVersionConsumerPoolStrategy extends ConsumerPoolStrategy {
     public CurrentVersionConsumerPoolStrategy() {
       this.consumerServiceForCurrentVersionAAWCLeader = consumerServiceConstructor.apply(
           serverConfig.getConsumerPoolSizeForCurrentVersionAAWCLeader(),
-          CURRENT_VERSION_AAWC_LEADER_STATS_SUFFIX);
+          CURRENT_VERSION_AAWC_LEADER_STATS_SUFFIX,
```

Comment:
Maybe we could use this ConsumerPoolType as suffix, I also planned to use it for consumption task thread name and task id str in my next PR, what do you think?

Reply:
yeah, we can do that. Or maybe just put the `suffix` definition into the enum definition.

Reply:
We could use the type to replace suffix, essentially we just want to pass the type information to KCS.

Reply:
We typically don't use the capital letters in the metric and I think we should follow the convention if possible.

Reply:
I think you most-recent update is what I meant.